docker/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM golang:1.10-alpine AS builder
 ARG SVC_NAME
+ARG GOARCH
```

Comment:
How is this used in Dockerfile? Looks like it is not.

Reply:
It is used in `&& make $SVC_NAME \` which needs this arguments.